astro/src/content/quickstarts/quickstart-javascript-express-web.mdx

```diff
@@ -0,0 +1,320 @@
+---
+title: Express
+description: Quickstart integration of an Express web application with FusionAuth
+navcategory: getting-started
+prerequisites: npm, node
+section: web
+technology: Express
+language: JavaScript
+icon: /img/icons/javascript.svg
+faIcon: fa-x
+color: fuchsia
+coderoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-javascript-express-web/main
+cta: EmailListCTA
+---
+import Aside from '/src/components/Aside.astro';
+import LoginAfter from '../../diagrams/quickstarts/login-after.astro';
+import LoginBefore from '../../diagrams/quickstarts/login-before.astro';
+import RemoteCode from '/src/components/RemoteCode.astro';
+
+
+In this quickstart you are going to build an application with Express and integrate it with FusionAuth.
+You'll be building it for [ChangeBank](https://www.youtube.com/watch?v=pkH-kD73QUM), a global leader in converting dollars into coins.
+It'll have areas reserved for users who have logged in as well as public facing sections.
+
+The docker compose file and source code for a complete application are available at
+https://github.com/FusionAuth/fusionauth-quickstart-javascript-express-web
+
+## Prerequisites
+* [Node LTS](https://nodejs.org/en/download)
```

Comment:
I think we should have a description here and also specify which version we used